src/pug/blog/report_10.pug

```diff
@@ -0,0 +1,59 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #10
+
+block content
+  h2 Progress report #10
+  +news_header('30 March 2023', 'report_9', '')
+
+  h3 Alphanumeric input research and testing
+  p We conducted research on typing methods on devices with limited inputs, and done a first round on internal alpha testing. We wrote the blog entry #[+a('alphanumeric_input') Alphanumeric input] with our preliminary findings.
+  p In the upcoming days we will release an Alpakka #[em Beta] firmware, with the purpose of gathering more feedback about some of these input methods.
+  p Other additions to the Beta will be a #[em shortcut for toggling the on-screen keyboard] both in Windows and Linux (including Steam Deck). And 4 additional predefined phrases for the quick chat functionality (for a total of 8).
+  p Expressing yourself with a controller was never so easy, stay tuned to #[+ax(org.discord) Discord] #[em #testing] channel!
+
+  hr
+  h3 Accessibility DB new entries
+  p Our lovely supporters in #[+ax(org.patreon) Patreon] voted for these games to be reviewed and included in the accessibility database:
+  ul
+    li: +a('/db/apex_legends') Apex Legends
+    li: +a('/db/control') Control
+    li: +a('/db/dishonored_2') Dishonored 2
+    li: +a('/db/gta_v') Grand Theft Auto V
+    li: +a('/db/skyrim') The Elder Scrolls V: Skyrim
+
+  hr
+  h3 Community mods taking off
+  p There was a significant amount of new contributions in the #[em #community-mods] channel on #[+ax(org.discord) Discord], and we cannot be happier, this is what Input Labs is about, infinite modding possibilities! These are just some cool examples:
```

Comment:
Can you add the refs (https://thangs.com/designer/Dennis%20The%20Menace/3d-model/Alpakka%20Taller%20ABXY%20%2B%20Dpad%20Buttons%20Mod%20%5BV1%5D-810164)